lightning/src/ln/channel.rs

```diff
@@ -646,6 +646,7 @@ mod state_flags {
 	pub const LOCAL_SHUTDOWN_SENT: u32 = 1 << 11;
 	pub const SHUTDOWN_COMPLETE: u32 = 1 << 12;
 	pub const WAITING_FOR_BATCH: u32 = 1 << 13;
+	pub const FUNDING_SIGNED: u32 = 1 << 14;
```

Comment:
`FUNDING_SIGNED` is added as part of `AwaitingChannelReadyFlags` and not a whole new thing